app/src/main/java/com/github/polypoly/app/base/menu/lobby/GameParameters.kt

```diff
@@ -18,21 +19,18 @@ data class GameParameters (
     val gameMode: GameMode = GameMode.RICHEST_PLAYER,
     val minimumNumberOfPlayers: Int = 3,
     val maximumNumberOfPlayers: Int = 7,
-    val roundDuration: Int = 360,
+    val roundDuration: Int = GameLobbyConstants.RoundDurations.getDefaultValue().toMinutes(),
     val maxRound: Int? = null,
     val gameMap: List<Zone> = LocationRepository.getZones(),
     val initialPlayerBalance: Int = 500,
 ) {
 
-    // TODO: define real value as this is useless
-    private val maxRoundHours = Int.MAX_VALUE
-
     init {
         if (minimumNumberOfPlayers <= 1)
             throw java.lang.IllegalArgumentException("At least 2 players are needed for a game (provided $minimumNumberOfPlayers)")
         if (maximumNumberOfPlayers < minimumNumberOfPlayers)
             throw java.lang.IllegalArgumentException("Maximum number of players $maximumNumberOfPlayers must be greater than the minimum number $minimumNumberOfPlayers")
-        if (roundDuration <= 0 || roundDuration >= maxRoundHours)
+        if (roundDuration <= 0 || roundDuration >= GameLobbyConstants.RoundDurations.values().last().toMinutes())
```

Comment:
Maybe instead of assuming the last value will always be the largest, create a method `RoundDurations.maxRound()` or something like that?

Reply:
This enum is supposed to be order so the last will always be the greatest. But I guess a maxRounds can be done to make things clearer